editor.planx.uk/src/@planx/components/Send/uniform/index.ts

```diff
@@ -2,9 +2,7 @@ import omit from "lodash/omit";
 
 import { Store } from "../../../../pages/FlowEditor/lib/store";
 import { getBOPSParams } from "../bops";
-import { findGeoJSON } from "../helpers";
-import { CSVData } from "../model";
-import { UniformPayload } from "./UniformPayload/model";
+import type { CSVData as PlanXExportData } from "../model";
```

Comment:
nit: I agree that `CSVData` isn't really a suitable name - we have a data structure that can be projected may ways. Rather than just rename on import, could we just rename the type?